From 3e361a9bdc5ec54f03ce37c61756028ac12970ba Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Erick=20P=C3=A9rez=20Castellanos?= <erick.red@gmail.com>
Date: Thu, 23 Apr 2015 02:03:12 -0400
Subject: [PATCH 01/16] types: added home to emails type

---
 src/contacts-types.vala | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/contacts-types.vala b/src/contacts-types.vala
index ec0ba82..b50d22a 100644
--- a/src/contacts-types.vala
+++ b/src/contacts-types.vala
@@ -304,12 +304,13 @@ public class Contacts.TypeSet : Object  {
   private const InitData[] email_data = {
     // List most specific first, always in upper case
     { N_("Personal"), { "PERSONAL" } },
+    { N_("Home"), { "HOME" } },
     { N_("Work"), { "WORK" } }
   };
   public static TypeSet email {
     get {
       string[] standard = {
-	"Personal", "Work"
+	"Personal", "Home", "Work"
       };
 
       if (_email == null) {
-- 
2.4.3

